Scripts/Core/Contexts/EditingContextManager.cs

```diff
@@ -24,26 +24,26 @@ sealed class EditingContextManager : MonoBehaviour
 
         internal static EditingContextManager s_Instance;
         static InputManager s_InputManager;
+        static List<IEditingContext> s_AvailableContexts;
+        static EditingContextManagerSettings s_Settings;
+        static UnityObject s_DefaultContext;
+        static IEditingContext s_CurrentContext;
 
-        EditingContextManagerSettings m_Settings = null;
-
-        List<IEditingContext> m_AvailableContexts;
-        string[] m_ContextNames = null;
+        string[] m_ContextNames;
```

Comment:
Making all of this static is a little worrying to me, but I haven't found any issues in testing.

Reply:
There's a need to access some context data before instantiation occurs.  I ran a ton of tests too, and couldn't find any issues.  If anyone wants to suggest, or craft a better solution, I'm open to any input/changes.